src/main/java/org/fundacionjala/sfdc/core/MyRuntimeException.java

```diff
@@ -0,0 +1,16 @@
+package org.fundacionjala.sfdc.core;
+
+/**
+ * Class that personalize the Runtime Exception.
+ */
+public class MyRuntimeException extends RuntimeException {
+
+    /**
+     * Constructor for the custom Runtime Exception.
+     *
+     * @param e exception event.
+     */
+    public MyRuntimeException(Exception e) {
```

Comment:
Please add a CustomException with a specific message
This class exception doesn't make sense